src/Cli.Tests/ConfigureOptionsTests.cs

```diff
@@ -109,6 +109,130 @@ public void TestAddDepthLimitForGraphQL()
             Assert.AreEqual(maxDepthLimit, config.Runtime.GraphQL.DepthLimit);
         }
 
+        /// <summary>
+        /// Tests that running "dab configure --runtime.graphql" on a config with variuos values results
+        /// in runtime.[DataTestMethod]
+        [DataRow(false, DisplayName = "Update enabled to be false for GraphQL.")]
+        [DataRow(true, DisplayName = "Update enabled to be true for GraphQL depth limit to -1.")]
+        [TestMethod]
+        public void TestUpdateEnabledForGraphQLSettings(bool updatedEnabledValue)
+        {
+            SetupFileSystemWithInitialConfig(INITIAL_CONFIG);
+
+            // Act: Attmepts to update enabled flag
+            ConfigureOptions options = new(
+                runtimeGraphQLEnabled: updatedEnabledValue,
+                config: TEST_RUNTIME_CONFIG_FILE
+            );
+            Assert.IsTrue(TryConfigureSettings(options, _runtimeConfigLoader!, _fileSystem!));
+
+            // Assert: Validate the Depth Limit is added
+            string updatedConfig = _fileSystem!.File.ReadAllText(TEST_RUNTIME_CONFIG_FILE);
+            Assert.IsTrue(RuntimeConfigLoader.TryParseConfig(updatedConfig, out RuntimeConfig? runtimeConfig));
+            Assert.IsNotNull(runtimeConfig.Runtime?.GraphQL?.Enabled);
+            Assert.AreEqual(updatedEnabledValue, runtimeConfig.Runtime.GraphQL.Enabled);
+        }
+
+        /// <summary>
+        /// Tests that running "dab configure --runtime.graphql" on a config with variuos values results
+        /// in runtime.[DataTestMethod]
```

Comment:
Please update the summary, it's exactly same as the previous one. It should specify what the test is doing.